Configurable drawer narrow layout

OrKoN opened the pull request:
using @custom-media. Already added the default custom-media (640px) to the default-theme. @vanthome please review

vanthome commented:
Have you set this new variable somewhere then? also it must be listed in the README.md under variables. And it must be set in the default theme...

OrKoN commented:
@vanthome yeah, I have added it to the default-theme https://github.com/vcl/default-theme/blob/master/index.styl#L507 and added to the doc https://github.com/vcl/default-theme/blob/master/README.md#custom-media.
